uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_absence-tab-page.scss

```diff
@@ -7,4 +7,8 @@
   .tab-pane {
     padding: 0 !important;
```

Comment:
Yes by default, the tab-pane is supposed to have the padding, but for absence tab page the padding is necessary. As this requirement is only for absence tab, we should not change original theme, instead we should just override the theme for absence tab pages. 

Also `!important` is necessary here because otherwise it wont work because of specificity rules. And its not a problem because this style is only meant for absence tab and not going to be reused. Also if we want to avoid `!important` we would need to add more classes or IDs which would be overkill and is not really necessary.